mm2src/lp_swap/taker_swap.rs

```diff
@@ -1330,6 +1338,13 @@ impl TakerSwap {
             },
         };
 
+        broadcast_transaction_message(
```

Comment:
I have noticed that it's better to `broadcast_transaction_message` also if `send_taker_refunds_payment` and other fns fail. Please make `send_taker_refunds_payment` and others to also return the generated transaction in case of error and call `broadcast_transaction_message` for it.

Reply:
`trait SwapOps` functions are (only `TransactionFut` returners) now calling `broadcast_transaction_message` right after generating transaction.

Reply:
> I'm a little confused sweat_smile Could you please tell where `trait SwapOps` functions call `broadcast_transaction_message`? I couldn't find it in the changes

Sorry, I couldn't explain correctly. I didn't call `broadcast_transaction_message`inside of the `trait SwapOps` functions, I call it in `maker_swap` and `taker_swap` after each `trait SwapOps` functions. But I guess Artem is pointing different thing than what I did here. I will update the changes today.